lib/ui/Auth/Login_Page/Auth.dart

```diff
@@ -219,13 +224,18 @@ class _AuthState extends State<Auth> {
                                       new BorderRadius.circular(30.0),
                                 ),
                                 onPressed: () {
-                                  //[TODO] ADD VALIDATOR HERE
+                                  if (!formAuthKey.currentState.validate()) {
+                                    ScaffoldMessenger.of(context)
```

Comment:
Use Fluttertoast (already in the pubsec.yaml), it looks aesthetic as well as goes with the UI